src/types/file.d.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Files
+ * Description: The immutable files API is available via the multipart/form-data Content-Type for image uploads.
+ * DOCS: https://documentation.elasticpath.com/commerce-cloud/docs/api/advanced/files/index.html
+ */
+import { core } from './core'
+
+export as namespace file
+
+export namespace file {
+  /**
+   * Core File Base Interface
+   * For custom flows, extend this interface
+   * DOCS: https://documentation.elasticpath.com/commerce-cloud/docs/api/advanced/files/index.html
+   */
+  export interface FileBase {
+    id?: string
+    type: string
+    link: {
+      href: string
+    }
+    file_name: string
+    mime_type: string
+    file_size: number
+    public: boolean
+    meta?: {
+      timestamps: {
+        created_at: string
+      }
+      dimensions: {
```

Comment:
didn't know this was a thing 😄

Reply:
This is very cool, I didn't know this was possible:
https://documentation.elasticpath.com/commerce-cloud/docs/api/advanced/files/index.html